app/templates/public/index.hbs

```diff
@@ -36,22 +36,6 @@
     </div>
     <div class="ui hidden divider"></div>
   {{/if}}
-  {{#if featuredSpeakers}}
-    <div class="featured-speakers" id="speakers">
-      <div class="row">
-        {{public/featured-speaker-list speakers=featuredSpeakers}}
-      </div>
-    </div>
-    <div class="ui hidden divider"></div>
-  {{/if}}
```

Comment:
Maybe we can display featured speakers on home page only? We have moved rest to speakers route.

Reply:
@ritikamotwani I think the featured speakers should be shown on the home page only. Featured speakers are like attractions of the event. It should be on the home page.

Reply:
@srv-twry +1